frontend/src/components/common/form/ParentSelect.tsx

```diff
@@ -57,13 +57,15 @@ export const ParentSelect: React.FC<IParentSelect> = ({
 
   useEffect(() => {
     if (!loaded) {
-      if (value?.value) {
-        setFieldValue(field, value);
-      } else if (value && !value.value) {
-        setFieldValue(field, option);
-      } else {
-        setFieldValue(field, option);
-        setLoaded(true);
+      if (value !== undefined) {
+        if (value?.value) {
```

Comment:
A test kept failing unless this was added, I can't remember what the reasoning was but @jdtoombs figured it out

Reply:
I don't really see why the if statement on 60 is required, unless the else statement shouldn't execute when value is undefined?

Reply:
I think it does throw an error threw the test if I remember correctly, I feel like this logic could be rearranged a bit better (from what I did originally). But I have a ticket for cleaning this component up/fixing warnings so I could do it then!

Reply:
`if (value?.value)` will return false if `value === undefined`, so I'm not sure why the need for `if (value !== undefined)`